packages/frontend/web/server/document.tsx

```diff
@@ -68,7 +68,11 @@ export const document = ({ data }: Props) => {
     const vendorJS = getDist('vendor.js');
     const polyfillIO =
         'https://assets.guim.co.uk/polyfill.io/v3/polyfill.min.js?rum=0&features=es6,es7,es2017,default-3.6,HTMLPictureElement,IntersectionObserver,IntersectionObserverEntry&flags=gated&callback=guardianPolyfilled&unknown=polyfill';
+    const commercialBundle = config.commercialUrl;
     const priorityScripts = [polyfillIO, vendorJS, bundleJS];
+    const preloadScripts = [
+        ...new Set([commercialBundle].concat(priorityScripts)),
+    ];
```

Comment:
Out of interest, we instantiate a `Set` here?

Reply:
This filters out duplicates in the event someone puts a script URL in both the arrays being concatenated, I don't know if it's likely to be honest! I could have gone with a `filter`/`reduce` to de-duping but I thought I'd try this approach instead.